src/box/lua/config/instance_config.lua

```diff
@@ -196,7 +196,7 @@ local function validate_uri_field(has_login_field, used_to_connect)
             if data:find(',') then
                 w.error('A single URI is expected, not a list of URIs')
             end
-            w.error('Unable to parse an URI: %s', err)
+            w.error('Unable to parse an URI: %s', err.message)
```

Comment:
If these verbose error messages are considered useful, why we discard it here? There is no word in the commit message that would explain what is specific of this particular error and why it should be handled differently from others.

Reply:
Sorry, I didn't pay attention that this is not a test harness. Your test expects a specific error message (while the new error serialized to a string contains a lot of other information besides the error message):
https://github.com/tarantool/tarantool/blob/94d2813841a649d5254974dd921aed01af1b24d4/test/config-luatest/instance_config_schema_test.lua#L812

I guess we should change it to an `assert_error_msg_contains` test.